src/assets/Doneth.sol.txt

```diff
@@ -58,8 +55,38 @@ contract Doneth {
         addMember(msg.sender, 1, true, _founderName);
     }
 
+    mapping(address => ERC20) public tokens;
+    address[] public tokenKeys;
```

Comment:
Is this used anywhere? I understand the rationale for having it, which is that it makes it possible to iterate over the keys, but it doesn't seem like you're not using it anywhere?

Reply:
ah true, think maybe under the check ERC20 balance function we should check to see if the token has a balance that it should be added to tokenKeys

Reply:
wonder if we could handle the ERC20 donations on our side as well? like combining the allow to transfer with the actual transfer....